.github/actions/terraform-apply/action.yml

```diff
@@ -33,6 +33,6 @@ runs:
         TF_VAR_aws_region: "${{ inputs.aws-region }}"
         TF_VAR_deployment_tag: "${{ steps.sha_short.outputs.sha_short }}"
         TF_VAR_env: "${{ inputs.env }}"
-        TF_VAR_git_commit_hash: "${{ github.sha }}"
+        TF_VAR_git_commit_hash: "${{ inputs.sha || github.sha }}"
```

Comment:
O kousek výš je to dobře: https://github.com/FigurePOS/github-actions/blob/459a71ecf0dbde640f98826b1a289b386308eecd/.github/actions/terraform-apply/action.yml#L24